testsuite/features/profiles/cloud_aws/Docker/authprofile/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Container used to test Content Management feature
+#
+# VERSION               1.0.0
+
+FROM ip-172-16-1-30.eu-central-1.compute.internal:5000/cucutest/systemsmanagement/uyuni/master/docker/containers/uyuni-master-testsuite
+MAINTAINER Michael Calmer "mc@suse.com"
```

Comment:
same as before, deprecated

Reply:
This is generated and such changes should be made in the template which did not change.
I think this should go into a separate PR